### PR TITLE
stat.c: clean up and symplifications

### PR DESCRIPTION
**Pull request by wtedeschi**
Exchanged repeated code with a loop, and created a function for max and min update.

### CODE REVIEW

**Comment by wtedeschi** [comment]
simplification on the stats handling.

File: src/stat.c [hunk 1]
```diff
@@ -51,37 +51,24 @@ int StatInit(bool logRtt)
...
-    GlobalStatistics[DCP_FRAME_TYPE].RoundTripMin = UINT64_MAX;
-    GlobalStatistics[LLDP_FRAME_TYPE].RoundTripMin = UINT64_MAX;
-    GlobalStatistics[UDP_HIGH_FRAME_TYPE].RoundTripMin = UINT64_MAX;
-    GlobalStatistics[UDP_LOW_FRAME_TYPE].RoundTripMin = UINT64_MAX;
-    GlobalStatistics[GENERICL2_FRAME_TYPE].RoundTripMin = UINT64_MAX;
+    for (int i = 0; i < NUM_FRAME_TYPES; i++)
+    {
+        struct Statistics *currentStats = &GlobalStatistics[i];
+        currentStats->RoundTripMin = UINT64_MAX;
+        currentStats = &GlobalStatisticsPerSec[i];
```
Comment (shifty91) [inline src/stat.c:69]: Doesn't compile, because GlobalStatisticsPerSec does not exist.

File: src/stat.c [hunk 2]
```diff
@@ -144,6 +126,12 @@ void StatFrameSent(enum StatFrameType frameType, uint64_t cycleNumber)
     stat->FramesSent++;
 }
 
+static inline void updateMinMax(uint64_t newValue, uint64_t *min, uint64_t *max)
```
Comment (shifty91) [inline src/stat.c:129]: Change function name to StatUpdateMinMax().

**Review by shifty91 — CHANGES_REQUESTED** [review]
Thanks a lot. Some minor comments only. Please, fix and i'll merge it.

**Pull request closed without merging**